src/S2EExecutionState.cpp

```diff
@@ -362,7 +362,7 @@ std::vector<ref<Expr>> S2EExecutionState::createConcolicArray(const std::string
     return result;
 }
 
-std::vector<ref<Expr>> S2EExecutionState::createSymbolicArray(const std::string &name, unsigned size,
+std::vector<ref<Expr>> S2EExecutionState::createSymbolicArray(const std::string &name, uintptr_t address, unsigned size,
```

Comment:
I would also pass the type of address here, similar to mem()->read()/write() apis which take an enum (virtual/physical/host).